src/libstore/globals.hh

```diff
@@ -902,6 +902,12 @@ public:
           resolves to a different location from that of the build machine. You
           can enable this setting if you are sure you're not going to do that.
         )"};
+
+    Setting<bool> offline{
+        this, false, "offline",
+        R"(
+          If set to `true` then Nix will avoid using any network resources.
```

Comment:
```suggestion
          If set to `true` then Nix will avoid using any network resources. In particular this will
          - Disable the substituters
          - Consider any previously downloaded file as up-to-date and refuse to download any new one
```

(well, maybe not the best wording, but I think explaining a bit more in details what’s going on is worth it)